config/example.yaml

```diff
@@ -4,4 +4,7 @@ example:
   param2: 30 # size_t
   param3: 1 # ros::Duration
   param4: [1, 1, 1, 3.14, 0, 0] # Eigen::Affine3d - x, y, z, roll, pitch, yaw
-  param5: [1.1, 2.2, 3.3, 4.4] # std::vector<double>
\ No newline at end of file
+  param5: [1.1, 2.2, 3.3, 4.4]  # std::vector<double>
+  param6: [1, 1, 1, 1, 0, 0, 0] # Eigen::Affine3d - x, y, z, qw, qz, qy, qz
+  param7: [1, 1, 1, 1, 0, 0, 0, 2, 2, 2, 1, 0, 0, 0] # EigenSTL::vector_Affine3d - [x, y, z, qw, qz, qy, qz, x, y, z, qw, qz, qy, qz]
```

Comment:
i don't like this approach, why not:

    param7:
        - [1, 1, 1, 1, 0, 0, 0]
        - [1, 1, 1, 1, 0, 0, 0]
        - [1, 1, 1, 1, 0, 0, 0]

output: EigenSTL::vector_Affine3d